scripts/production/backup_master.pl

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env perl
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+=head1 NAME
+
+backup_master.pl
+
+=head1 DESCRIPTION
+
+This script is a wrapper around sudo and mysqldump to dump the master database,
+taking its location from a production_reg_conf file.
+
+=head1 SYNOPSIS
+
+  perl backup_master.pl --reg_conf $COMPARA_REG_PATH
+
+  perl backup_master.pl $COMPARA_REG compara_master --label "pre100"
```

Comment:
```suggestion
    perl backup_master.pl --reg_conf $COMPARA_REG_PATH

    perl backup_master.pl $COMPARA_REG compara_master --label "pre100"
```

Reply:
FYI 1 space is enough to trigger a "verbatim" paragraph: https://perldoc.perl.org/perlpod.html#Verbatim-Paragraph